pkgs/by-name/cp/cppitertools/package.nix

```
@@ -0,0 +1,97 @@
+{
+  lib,
+  stdenv,
+  cmake,
+  boost,
+  catch2,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "cppitertools";
+  version = "2.1";
+
+  src = fetchFromGitHub {
+    owner = "ryanhaining";
+    repo = "cppitertools";
+    rev = "v${finalAttrs.version}";
```

```suggestion
    rev = "refs/tags/v${finalAttrs.version}";
```

sometimes update bots get confused otherwise